dwave/optimization/model.pyx

```diff
@@ -1358,17 +1358,32 @@ cdef class Symbol:
             inc(it)
 
     def maybe_equals(self, other):
-        """Compare to another node.
+        """Compare to another node. This method exists because a complete equality test can be expensive.
```

Comment:
```suggestion
        """Compare to another node.
        
        This method exists because a complete equality test can be expensive.
```
IMO this makes sense to include in the body rather than in the summary line. See [PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings).